src/routers/v2/DelaysRouter.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import ApplicationRouter from '../../appdev/ApplicationRouter';
+import RealtimeFeedUtils from '../../utils/RealtimeFeedUtils';
+
+class DelaysRouter extends ApplicationRouter<Object> {
+  constructor() {
+    super(['POST']);
+  }
+
+  getPath(): string {
+    return '/delays/';
+  }
+
+  async content(req): Promise<any> {
+    const delays = req.body.data;
+    const rtf = await RealtimeFeedUtils.fetchRTF();
+    delays.map(async (stopAndTripID) => {
```

Comment:
i think `stopAndTripID` could be either be renamed something else more indicative of the actual function -- `tripData` or something along those lines -- or just directly destructured -- `async ({stopID, tripID}) => {`.